Zend/tests/closure_const_expr/disallows_non_static.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Disallows using variables.
+--FILE--
+<?php
+
+$foo = "bar";
```

Comment:
looks like this `$foo` variable is redundant in this test

Reply:
Thanks good catch, this was bad copy-and-paste. I've rewritten the test entirely to something where the difference between static and non-static would actually be relevant: 55d46874cbf9b98dc24eda318392192a46e4dde9.